From 02ab23d27cfd32b8be513956bee63e0e5bd07f6f Mon Sep 17 00:00:00 2001
From: Lijuan Gao <quic_lijuang@quicinc.com>
Date: Mon, 23 Sep 2024 18:07:10 +0800
Subject: [PATCH 56/65] dt-bindings: remoteproc: qcom,sm8150-pas: Document
 QCS615 remoteproc

Document the ADSP and CDSP Peripheral Authentication Service on
QCS615 platform.

Signed-off-by: Lijuan Gao <quic_lijuang@quicinc.com>
---
 .../devicetree/bindings/remoteproc/qcom,sm8150-pas.yaml       | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/remoteproc/qcom,sm8150-pas.yaml b/Documentation/devicetree/bindings/remoteproc/qcom,sm8150-pas.yaml
index d67386c50fa4..e868c76c0dc0 100644
--- a/Documentation/devicetree/bindings/remoteproc/qcom,sm8150-pas.yaml
+++ b/Documentation/devicetree/bindings/remoteproc/qcom,sm8150-pas.yaml
@@ -60,6 +60,8 @@ allOf:
       properties:
         compatible:
           enum:
+            - qcom,qcs615-adsp-pas
+            - qcom,qcs615-cdsp-pas
             - qcom,sm8150-adsp-pas
             - qcom,sm8150-cdsp-pas
             - qcom,sm8150-slpi-pas
@@ -83,6 +85,8 @@ allOf:
       properties:
         compatible:
           enum:
+            - qcom,qcs615-adsp-pas
+            - qcom,qcs615-cdsp-pas
             - qcom,sm8150-adsp-pas
             - qcom,sm8150-cdsp-pas
             - qcom,sm8250-cdsp-pas
-- 
2.34.1

